cookbook/integration_crew_ai.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>this needs 2 sentences on how Langfuse OpenAI Integration and observe() decorator are used together, how does this work?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/546/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>add links to: crewai, and the two integration docs pages</p>